pkg/fanal/analyzer/all/import.go

```diff
@@ -12,6 +12,7 @@ import (
 	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dart/pub"
 	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dotnet/deps"
 	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dotnet/nuget"
+	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dotnet/packagesprops"
```

Comment:
> It seems packagesprops logic is not registered somewhere.

You needed to add `packagesprops` package here to enable parsing of `*Packages.props` files.